lambdas/models/auth_policy.py

```diff
@@ -36,8 +36,8 @@ class AuthPolicy(object):
     region = os.getenv("AWS_DEFAULT_REGION", "eu-west-2")
     stage = "dev"
 
-    def __init__(self, principal, awsAccountId):
-        self.awsAccountId = awsAccountId
+    def __init__(self, principal, aws_account_id):
+        self.awsAccountId = aws_account_id
         self.principalId = principal
```

Comment:
I know this code existed before this PR, but there's camel case variables in this class that should probably be changed to snake case as part of the refactor

Reply:
Done